src/Tasks/Backup/FileSelection.php

```diff
@@ -168,4 +168,20 @@ protected function sanitize($paths): Collection
                 return $path === false;
             });
     }
+
+    /**
+     * @param string $path
+     *
+     * @return array
+     */
+    protected function getMatchingPaths(string $path): array
+    {
+        if (strpos($path, '*') !== false && defined('GLOB_BRACE')) {
+            $result = glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE);
+        } else {
```

Comment:
Please refactor this else away by using early returns

Reply:
else removed